script/process.py

```diff
@@ -108,7 +108,7 @@ def process():
 def process_file(path: pathlib.Path):
     total = UpdateRecord()
 
-    with path.open("r") as file:
+    with path.open("r", encoding="utf-8-sig") as file:
```

Comment:
It depends on how you saved your CSV file. We should leave this to auto-detect.
```suggestion
    with path.open("r") as file:
```